freezeyt/freezing.py

```diff
@@ -60,7 +60,10 @@ def url_to_filename(base, url, hostname='localhost', port=8000, path='/'):
     return base / url_path.lstrip('/')
 
 
-def freeze(app, path, prefix='http://localhost:8000/', extra_pages=()):
+def freeze( app, path,
+            prefix='http://localhost:8000/',
+            extra_pages=(),
+            extra_files={}):
```

Comment:
Slovník se dá měnit, takže to není dobrá výchozí hodnota argumentu. I když to teď nevadí, v budoucnu se může stát že někdo zapomene, že tenhle slovník se nesmí měnit (protože je to výchozí hodnota `extra_files` i pro všechny další volání `freeze`).
Je lepší použít `None` a dole pak při `None` celý cyklus přeskočit.

Reply:
Říkal jsem si no, že je to mutable